cuqi/problem/_problem.py

```diff
@@ -587,5 +629,87 @@ def _check_posterior(self, prior_type=None, likelihood_type=None, model_type=Non
 
         return L and P and M and D and G
 
+    def _sampleGibbs(self, Ns, callback=None):
+        """ This is a helper function for sampling from the posterior using Gibbs sampler. """
+        print("Using Gibbs sampler")
+        print("burn-in: 20%")
```

Comment:
Use variable for percentage here and in line 645 instead of hardcoding.

Reply:
Good suggestion :)